Zilon.Core/Zilon.Tournament.ApiGate/Launcher/LauncherService.cs

```diff
@@ -38,40 +38,42 @@ protected override Task ExecuteAsync(CancellationToken stoppingToken)
                     {
                         try
                         {
-                            using var process = new Process();
-
-                            process.StartInfo = new ProcessStartInfo
+                            using (var process = new Process())
                             {
-                                FileName = $"{appPath}Zilon.BotMassLauncher.exe",
-                                UseShellExecute = false,
-                                CreateNoWindow = true,
-                                Arguments = $"parallel=10 mode={mode} botCatalog={botInfo.Catalog} botAssembly={botInfo.Assembly} env=\"{appPath}Zilon.BotEnvironment.exe\" launchCount=1000 output=\"{outputCatalog}\" schemeCatalogPath=\"{schemeCatalogPath}\"",
-                                RedirectStandardOutput = true,
-                                RedirectStandardError = true
-                            };
-
-                            process.Start();
-
-                            var output = process.StandardOutput.ReadToEnd();
-                            var error = process.StandardError.ReadToEnd();
-
-                            // Один бот в массланчере в одном режиме может работать 1 час максимум
-                            process.WaitForExit((int)(3.6 * 1000_000));
+                                process.StartInfo = new ProcessStartInfo
+                                {
+                                    FileName = $"{appPath}Zilon.BotMassLauncher.exe",
+                                    UseShellExecute = false,
+                                    CreateNoWindow = true,
+                                    Arguments = $"parallel=10 mode={mode} botCatalog={botInfo.Catalog} botAssembly={botInfo.Assembly} env=\"{appPath}Zilon.BotEnvironment.exe\" launchCount=1000 output=\"{outputCatalog}\" schemeCatalogPath=\"{schemeCatalogPath}\"",
+                                    RedirectStandardOutput = true,
+                                    RedirectStandardError = true
+                                };
+
+                                process.Start();
+
+                                var output = process.StandardOutput.ReadToEnd();
+                                var error = process.StandardError.ReadToEnd();
+
+                                // Один бот в массланчере в одном режиме может работать 1 час максимум
+                                process.WaitForExit((int)(3.6 * 1000_000));
+                            }
                         }
-                        catch (Exception e)
+                        catch (Exception exception)
                         {
+                            Console.WriteLine($"[X] {exception}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5232990)